azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
 
 
 class IonQAriaQirBackend(IonQQirBackendBase):
```

Comment:
We need to configure a new backend with different configuration

Reply:
n_qubits should be 35 per https://dev.azure.com/AzureQuantum-Partners/Partners/_git/ionq-documents/pullrequest/474?_a=files

Reply:
Fixed this.